pkg/kyma/cmd/provision/minikube/hosts_darwin.go

```diff
@@ -11,18 +16,65 @@ const hostsFile = "/etc/hosts"
 
 const defaultVMDriver = vmDriverHyperkit
 
+func isWithSudo() bool {
+	val := os.Getenv("SUDO_UID")
+	if val != "" {
+		return true
+	}
+	return false
+}
+
+func promptUser() bool {
+	for {
+		fmt.Print("Type [y/n]: ")
+		var res string
+		_, err := fmt.Scanf("%s", &res)
```

Comment:
Since `res` is already declared outside, this can be expressed as:
```
if _, err := fmt.Scanf("%s", &res); err != nil {
  return false
}
...
```
Saves 1 line and err only exists in the scope of that if statement.

Reply:
Fixed